dpctl/tensor/libtensor/include/kernels/clip.hpp

```diff
@@ -216,22 +216,26 @@ sycl::event clip_contig_impl(sycl::queue &q,
         {
             constexpr bool enable_sg_loadstore = true;
             using KernelName = clip_contig_kernel<T, vec_sz, n_vecs>;
+            using Impl =
+                ClipContigFunctor<T, vec_sz, n_vecs, enable_sg_loadstore>;
+            static_assert(sycl::is_device_copyable_v<Impl>);
```

Comment:
It may make sense to add these assertions everywhere that a kernel functor is used, but that can be deferred to a subsequent PR.

Reply:
Actually no, I should remove this one too. For example, `sycl::local_accessor<T, dim>` is not device copyable, so all functors that work with local accessors won't be. 

I started adding these assertions and realized they were a mistake. Will remove this one too. Indexers should be device copyable though. Basic types are by out of the box.